app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,7 @@
         android:name=".application.TreeTrackerApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@mipmap/ic_launcher"
```

Comment:
Why was the icon shape here changed?

Reply:
For some reason, I don't have the ic_launcher_round in my checkout files. I will double check this again.